include/RED4ext/ISerializable.hpp

```diff
@@ -46,10 +46,9 @@ struct ISerializable
     virtual void* sub_C8(void* a1);                                                     // C8
     virtual bool CanBeDestructed();                                                     // D0
 
-    WeakHandle<ISerializable> ref; // 00
-    int64_t unk18;                 // 18
-    int64_t unk20;                 // 20
-    int64_t unk28;                 // 28
+    WeakHandle<ISerializable> ref;   // 00 - Initialized in Handle ctor
+    WeakHandle<ISerializable> unk18; // 18
+    uint64_t unk28;                  // 28 - Incremental ID set in ISerializable ctor, can be zero
```

Comment:
What is this incremental ID used for?

Reply:
It's used as a key in the global serializable storage. The storage is there, but empty in the release. I think they only use it in dev environment. There's also a class that takes a snapshot of all known objects using this storage. The snapshot is created before loading scripts and restored after that. I think it's meant for hot reloading.

Reply:
Thanks!